operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -758,6 +766,14 @@ alertmanager:
         # If set to "-", then use `storageClassName: ""`, which disables dynamic provisioning
         # If undefined or set to null (the default), then fall back to the value of `alertmanager.persistentVolume.storageClass`.
         storageClass: null
+        # -- noDownscale adds a label that can be used by the rollout-operator as documented in the rollout-operator repo: https://github.com/grafana/rollout-operator#webhooks
+        noDownscale: true
+        # -- downscaleLeader is an Annotation used by the rollout-operator
+        # If defined, downscaleLeader: <downscaleLeader>
+        # If undefined or set to null (the default), no Annotation is set
+        downscaleLeader: null
```

Comment:
would be great if we can set the downscale leader automatically in the zonal templates. That way automatica downscaling can be just a single boolean toggle for the ingester/store-gateway which automatically sets up downscale leader and prepare downscale for the 3 zones.

But i think this PR has been lingering for long enough to try to slow it down further. happy to **not** include this change in the PR if you want.

Reply:
having said that - i think having `noDownscale: false` would be a safer change because it will mean folks can keep running the chart as they used to. If we get automatic downscaling automatically set up like I suggested above, then I think it's viable to have it on by default. 

I think it makes sense to try to minimize the added manual effort for folks after they upgrade to a chart version which contains this PR

Reply:
i think that changing `noDownscale: false` makes sense. and as you mentioned the PR has been lingering for a while and i just don't have the bandwidth to deal with it.